CONTRIBUTING.md

```diff
@@ -15,6 +15,10 @@ source venv/bin/activate
 pip install -e .'[dev]'
 ```
 
+##### NOTE: You might run into issues where you have a local install and are trying to work with a plugin pinned to a specific version.
```

Comment:
`#` is for titles, not notes.
For notes do this use the asterisks. (Look at other parts of the docs with the bolded notes part.)